src/openrct2/drawing/Drawing.String.cpp

```diff
@@ -1075,45 +1075,28 @@ void GfxDrawStringWithYOffsets(
     dpi.lastStringPos = { info.x, info.y };
 }
 
-void ShortenPath(utf8* buffer, size_t bufferSize, const utf8* path, int32_t availableWidth, FontStyle fontStyle)
+u8string ShortenPath(const u8string& path, int32_t availableWidth, FontStyle fontStyle)
```

Comment:
Any blockers preventing from making this a `u8string_view`? Maybe `GfxGetStringWidth` isn't ready for that?

Reply:
Not exactly a blocker but we should turn things to string_view once its all done.

Reply:
Don't make the same mistake I did, and only do them small chunks at a time. Makes it much easier to move to `std::string_view`.